receiver/purefbreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,24 @@
+receivers:
+  purefb:
+    endpoint: http://172.31.60.207:9491/metrics
+
+  purefb/with_custom_intervals:
+    endpoint: http://172.31.60.207:9491/metrics
+    arrays:
+      - address: gse-fb01
```

Comment:
Please remove gse prefix rom these host names as they’re internal. Can be set just to “fb01”